WordPress/WordPressTest/WPUserAgentTests.m

```diff
@@ -0,0 +1,67 @@
+//
+//  WPUserAgentTests.m
+//  WordPress
+//
+//  Created by Diego E. Rey Mendez on 3/17/15.
+//  Copyright (c) 2015 WordPress. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+
+#import "WPUserAgent.h"
+
+static NSString* const WPUserAgentKeyUserAgent = @"UserAgent";
```

Comment:
Maybe instead of duplicating these three keys you could make the constants in https://github.com/wordpress-mobile/WordPress-iOS/pull/3421/files#diff-8cce8cde5c9de1e79dab949ad54b2951R3 external?